creator-node/src/RehydrateIpfsQueue.js

```diff
@@ -23,34 +22,13 @@ class RehydrateIpfsQueue {
     )
 
     // Most errors in the rehydrate calls will be caught; this try/catch is to catch unexpected errors
-
-    this.queue.process(PROCESS_NAMES.rehydrate_file, config.get('rehydrateMaxConcurrency'), async (job, done) => {
-      const { multihash, storagePath, filename, logContext } = job.data
-
-      this.logStatus(logContext, `Processing a rehydrateIpfsFromFsIfNecessary task for ${multihash}`)
-      try {
-        await rehydrateIpfsFromFsIfNecessary(multihash, storagePath, logContext, filename)
-        done()
-      } catch (e) {
-        this.logError(logContext, `Problem with processing a rehydrateIpfsFromFsIfNecessary task for ${multihash}: ${e}`)
-        done(e)
-      }
-    })
-
-    this.queue.process(PROCESS_NAMES.rehydrate_dir, config.get('rehydrateMaxConcurrency'), async (job, done) => {
-      const { multihash, logContext } = job.data
-      this.logStatus(logContext, `Processing a rehydrateIpfsDirFromFsIfNecessary task for ${multihash}`)
-      try {
-        await rehydrateIpfsDirFromFsIfNecessary(multihash, logContext)
-        done()
-      } catch (e) {
-        this.logError(logContext, `Problem with processing a rehydrateIpfsDirFromFsIfNecessary task for ${multihash}: ${e}`)
-        done(e)
-      }
-    })
+    this.queue.process(PROCESS_NAMES.rehydrate_file, MAX_CONCURRENCY, `${__dirname}/rehydrateIpfsTasks.js`)
```

Comment:
can we delete some of these other tasks like rehydrate_file and rehydrate_dir?

Reply:
I don't think we should. We still rehydrate ipfs by CID in other areas of the code base like https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/routes/files.js#L120

Reply:
Unless we decide that we ultimately do not want to rehydrate in other areas anymore

Reply:
@vicky-g is that still necessary? given what you have here, shouldn't any user rehydrate cover a specific CID?

Reply:
discussed offline -- will keep rehydrate by dir and file if problems arise as they provide specific rehydration for moments we need specific cids